tests/Fixer/LanguageConstruct/IsNullFixerTest.php

```diff
@@ -204,6 +205,36 @@ public function provideYodaFixCases()
                 '<?php $result = ((null === $a) <> false); ?>',
                 '<?php $result = (is_null($a) <> false); ?>',
             ],
+
+            // check with logical operator
```

Comment:
The next test case does not contain any logical operator, shouldn't the comment be after it?